Lighthouse Camera/UI/Main/TTSHelper.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TTSHelper.swift
```

Comment:
I'm not convinced we should even speak of FTE as a thing - we don't have a demonstrated need for it yet. We talked about possibly having additional info - like verbose labels/descriptions which become more terse once the user has the hang of it.